docs/src/main/sphinx/release/release-375.md

```diff
@@ -0,0 +1,81 @@
+# Release 375 (25 Mar 2022)
+
+## General
+
+* Change behavior of `ALTER TABLE qualified_name TO unqualified_name` to leave
+  the table in the schema. This is backwards-incompatible behavioral change. ({issue}`11282`)
+* Disallow table redirections for the `GRANT`, `REVOKE`, and
+  `SET TABLE AUTHORIZATION` table tasks. ({issue}`11302`)
+* Improve performance of queries that contain filter predicates on aggregation
+  results. ({issue}`11469`)
+* Fix `SHOW CREATE TABLE` to show actual table name in case of table
+  redirections, so that the statement can be used to recreate the table. ({issue}`11604`)
+* Fix scheduling for non-remotely accessible splits in fault-toleration execution. ({issue}`11581`)
+* Fix incorrect `is_nullable` value in `information_schema.columns` table. ({issue}`11613`)
+
+## JDBC driver
+
+* Allow `PreparedStatement.close()` to be called multiple times. ({issue}`11620`)
+* Fix incorrect `is_nullable` value in `DatabaseMetaData.getColumns()` method. ({issue}`11613`)
+
+## Cassandra connector
+
+* Return `0` instead of row count in completed bytes. ({issue}`11644`)
+
+## Delta Lake connector
+
+* Add access control to `drop_extended_stats` and `vacuum` procedures. ({issue}`11633`)
+* Fix incorrect query results when previous query coincides with drop table. ({issue}`11562`)
```

Comment:
What does “coincides with” mean in this context. Not sure I understand what the item is trying to convey

Reply:
Not sure either .. seems to have to to with the transaction log from looking at the PR ..